setup.cfg

```diff
@@ -19,44 +19,16 @@ setup_requires =
     setuptools >= 38.3.0
     setuptools_scm
 install_requires =
-    cartopy
-    cftime == 1.0.1
-    cf_units
-    clize
-    dask
-    netCDF4
-    numpy
-    python-dateutil
-    pytz
-    scipy >= 1.3.0, < 1.4.0
+# Note: only minimal requirements are included here for development usage.
```

Comment:
Will a pip install give developers something where all un-skipable tests will pass?
I just wondered what purpose a pip install still serves i.e. is it worth still supporting or is it something we don't want people to use or have to support?

Reply:
The intention here was not to support `pip install` as a general method for installing IMPROVER. However, pip is needed for building conda packages, so we can't completely remove `setup.cfg`. Removing the `install_requires` section would make this clearer than the current situation of having a not-really-working specification.

Additionally, my understanding is that `pip install --editable` (or `-e`) is the preferred method for editable development installation when using conda. `conda develop` exists but has issues eg. https://github.com/conda/conda-build/issues/4251 and https://github.com/conda/conda-build/issues/1992#issuecomment-525777055